.cirrus.yml

```diff
@@ -64,10 +64,10 @@ task:
     FILE_ENV: "./ci/test/00_setup_env_native_qt5.sh"
 
 task:
-  name: 'x86_64 Linux  [GOAL: install]  [focal]  [depends, sanitizers: thread (TSan), no gui]'
+  name: 'x86_64 Linux  [GOAL: install]  [lunar]  [depends, sanitizers: thread (TSan), no gui]'
   << : *GLOBAL_TASK_TEMPLATE
   container:
-    image: ubuntu:focal
+    image: ubuntu:lunar
```

Comment:
I think NACK here? https://fridge.ubuntu.com/2023/12/15/ubuntu-23-04-lunar-lobster-reaches-end-of-life-on-january-25-2024/

Lunar is past EOL

Reply:
we do not use cirrus, so, that's not actually important which exactly dist is specified here

Reply:
Ah true 🙈

Reply:
this line is matter: https://github.com/dashpay/dash/pull/5867/files#diff-0fa5cb5863297feef0a044ed7983d9804452cdc83a6382fec3e1e127b9915a30R141

and we use custom repo with llvm -> doesn't matter which exactly version is provided by ubuntu

Reply:
Is there a reason why we don't have to bump our ubuntu:focal in contrib/containers/ci/Dockerfile?